### PR TITLE
added theme extractor and added table properties

### PR DESCRIPTION
**Pull request by akhilesh-fincity**

### CODE REVIEW

File: ui-app/client/src/components/util/properties.ts [hunk 1]
```diff
@@ -156,7 +156,7 @@ const COMPONENT_STYLE_GROUPS: { [key: string]: Array<string> } = {
 	margin: ['marginBottom', 'marginLeft', 'marginRight', 'marginTop'],
 	opacity: ['opacity'],
 	outline: ['outlineColor', 'outlineOffset', 'outlineStyle', 'outlineWidth'],
-	padding: ['paddingBottom', 'paddingLeft', 'paddingRight', 'paddingTop'],
+	padding: ['paddingBottom', 'paddingLeft', 'paddingRight', 'paddingTop', 'padding'],
```
Comment (raja-avinash) [inline ui-app/client/src/components/util/properties.ts:159]: remove padding

**Review by raja-avinash — APPROVED** [review]
LGTM

File: ui-app/client/src/components/util/properties.ts (hunk 1 above)
Comment (serialcoder13) [reply ui-app/client/src/components/util/properties.ts]: It is removed I see.

**Review by serialcoder13 — APPROVED** [review]
LGTM

**Pull request merged**